website/client-old/js/services/socialServices.js

```diff
@@ -34,8 +36,27 @@
       }
     }
 
+    hello.init({
+      facebook : window.env.FACEBOOK_KEY,
+      google : window.env.GOOGLE_CLIENT_ID
+    });
+
+    function socialLogin(network){
+      hello(network).login({scope:['email', 'profile']}).then(function(auth){
```

Comment:
I'm getting an out of scope error when I register with Facebook. Should this be: `hello(network).login({scope:'email'})` instead?